client/src/components/PostBody/PostBody.css

```diff
@@ -5,46 +5,46 @@
   color: var(--black);
 }
 
-.post-body-top {
+.PostBody--top {
```

Comment:
Shouldn't these be `PostBody__top`?

Reply:
fixed this in the other branch and just merged those changes into this branch